best-practices/javascript_debugging.md

```diff
@@ -0,0 +1,8 @@
+# Debugging in Javascript
+
+If you'd like to use a stepwise debugging in javascript, consider using the following procedure when working in VSCode.
```

Comment:
Should this read "If you'd like use stepwise..."? Other thank, :+1:

Reply:
```suggestion
If you'd like to use stepwise debugging in javascript, consider using the following procedure when working in VSCode.
```